z7/remove_repeating.cpp

```diff
@@ -0,0 +1,15 @@
+#include "remove_repeating.hpp"
+
+#include <algorithm>
+
+void removeDuplicates(std::vector<std::string>& str) {
```

Comment:
this function should take std::string as an argument, not std::vector<std::string>.
In the first step you should probably convert string to the vector of words